actions/access_keys/create.go

```diff
@@ -18,9 +18,9 @@ import (
 // @Param		metadata query string false "metadata"
```

Comment:
General remark:
I think that swagger should be regenerated.
![image](https://github.com/BuxOrg/bux-server/assets/48433067/3ceb2291-4d1c-4548-ae41-333f434949ea)

I can help you with that.